ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -351,6 +359,9 @@ exampleTxBodyAllegra = exampleTxBodyMA exampleCoin
 exampleTxBodyMary :: Core.TxBody StandardMary
 exampleTxBodyMary = exampleTxBodyMA exampleMultiAssetValue
 
+exampleTxBodyAlonzo :: Core.TxBody StandardAlonzo
```

Comment:
FYI, this will be a type error once we have the real Alonzo era tag type: Alonzo uses some different types for its transactions (I cannot recall off the top of my head, unfortunately). Just a heads up.